src/config/env.js

```diff
@@ -0,0 +1,30 @@
+require("dotenv").config();
```

Comment:
@shrijan-siris probably it is a good idea to have separate .env files for dev, qa and prod (as done in our company website using gatsby)
```
require('dotenv').config({
	path: `.env.${process.env.NODE_ENV}`,
})
```

Reply:
we should try to remove if/else , switch statements coded for each environment

Reply:
great